prac_08/box_layout_demo.py

```diff
@@ -0,0 +1,21 @@
+# Adrianto Hartono
+# 14347930
+
+from kivy.app import App
+from kivy.lang import Builder
+
+class BoxLayoutDemo(App):
+    def build(self):
+        self.title = "Box Layout Demo"
+        self.root = Builder.load_file('box_layout.kv')
+        return self.root
+
+    def handle_greet(self):
+        print("test")
+        self.root.ids.output_label.text = f"Hello {self.root.ids.input_name.text}"
+
+    def clear_input(self):
+        self.root.ids.input_name.text = f""
```

Comment:
You don't need to have the f in front of the ""

Reply:
Thank you for the feedback, noteed🙏